### ISSUE TITLE
About the loss in the training process.

### ISSUE THREAD

**Issue by IssacCyj** [amdegroot/ssd.pytorch#96]
I have trained the model in VOC2007+2012 and also adapted it to other datasets. I've try 300, and 640, and some different settings as well. However, the final loss is always around 2.x or 3.x and keep in this level. What is your final loss when training? I am confused with this problem for several days and didn't found the reason. Plz give me some hints!

**Comment by IssacCyj** [comment]
Oh it might not reflects a worse result, here's mine on voc2007 test set.
`AP for aeroplane = 0.8015
AP for bicycle = 0.8290
AP for bird = 0.7554
AP for boat = 0.7035
AP for bottle = 0.4985
AP for bus = 0.8434
AP for car = 0.8588
AP for cat = 0.8802
AP for chair = 0.6105
AP for cow = 0.8121
AP for diningtable = 0.7808
AP for dog = 0.8443
AP for horse = 0.8671
AP for motorbike = 0.8353
AP for person = 0.7908
AP for pottedplant = 0.5000
AP for sheep = 0.7574
AP for sofa = 0.7915
AP for train = 0.8579
AP for tvmonitor = 0.7613
Mean AP = 0.7690`

FYI: the final location loss is around 1 and confidence loss is around 2

**Comment by IssacCyj** [comment]
I just followed the original training steps and got the results. Maybe it was caused by different initialization?

**Comment by OxInsky** [comment]
Thanks!I solved it, mistaking to init the 'lr '=1e-4,so got a worse result！

**Comment by Ai-is-light** [comment]
@OxInsky @IssacCyj 
 when I set the original lr=1e-3, after 50-100 iterations, it will get NaN loss; while setting 'lr'=1e-4, it will get the decreasing loss.
are looking forward 7fdcf168f8f2ef4aca47cb4f69757ac1ce05700e8814e098
